### PR TITLE
Clickable names

### PR DESCRIPTION
**Pull request by threezool**
Changed the names in the persons list to be clickable in regards to #59

### CODE REVIEW

**Comment by warpech** [comment]
@Joltrix could you please review?

File: src/People/ViewModels/PersonsPage.json.cs [hunk 1]
```diff
@@ -76,6 +76,8 @@ void Handle(Input.EntriesPerPage action)
         [PersonsPage_json.Persons]
         partial class PersonsPersonPage : Json, IBound<Person>
         {
+            public string ViewUrl => $"/people/persons/{this.Data.Key}";
```
Comment (Joltrix) [inline src/People/ViewModels/PersonsPage.json.cs]: @warpech What do you think about this? I mean, it does the same as the string.Format , should this be changed to "fit the standard": use string.Format like in all other places, or should we just leave it as it is?

Comment (warpech) [inline src/People/ViewModels/PersonsPage.json.cs]: I think we should have the same code in all places. But actually interpolated string looks better to me. Maybe we should change `string.Format` to an interpolated string in the whole app, WDYT?

Comment (Joltrix) [inline src/People/ViewModels/PersonsPage.json.cs]: Yeah, So, it's either replacing all string.Formats with interpolated strings, or the other way around :P.

hmm, I'm not really sure, the only thing I don't like with the interpolated strings is the "$", but except for that, its more understandable than {0}, and then having to check in the "parameters" for the value of {0}.

But after researching it a bit online, string interpolation seems to be the preferred way, So I don't mind switching to interpolated strings :)

Comment (per-samuelsson) [inline src/People/ViewModels/PersonsPage.json.cs]: FWIW, regarding

> Maybe we should change string.Format to an interpolated string in the whole app, WDYT?

interpolated strings will very likely become the future C# standard, and was introduced to make `string.Format`-pattern less noisy. For hosting code, we are going that way for sure.

Comment (warpech) [inline src/People/ViewModels/PersonsPage.json.cs]: Decided then. @threezool can you please change all other occurrences of `string.Format` in this app to interpolated strings?

Comment (threezool) [inline src/People/ViewModels/PersonsPage.json.cs]: Will do

**Comment by warpech** [comment]
Ping @threezool. Any updates regarding this?

**Comment by warpech** [comment]
LGTM! @threezool please merge to `develop`.

**Comment by warpech** [comment]
Great, thanks!

**Pull request merged**